zerver/lib/channel_folders.py

```diff
@@ -33,7 +33,7 @@ def check_channel_folder_name(name: str, realm: Realm) -> None:
         )
 
     if ChannelFolder.objects.filter(name__iexact=name, realm=realm).exists():
-        raise JsonableError(_("Channel folder '{name}' already exists").format(name=name))
+        raise JsonableError(_("Channel folder name already in use"))
```

Comment:
Changed the error message to be same as the stream case. Was not sure what name to show in error if we want to include the name - the name of the folder which exists or the new name requested.